frontend/app/controllers/reviews/show.js

```diff
@@ -1,22 +1,26 @@
 import Ember from 'ember';
 
-export default Ember.ObjectController.extend({
-  coverImageStyle: function () {
-    return "background-image: url(" + this.get('user.coverImageUrl') + ")";
-  }.property('user.coverUrl'),
-  writtenBySelf: function () {
-    return this.get('currentUser.isSignedIn') && this.get('currentUser.id') === this.get('user.id');
-  }.property('user.id'),
-  ratingFirstDigit: function () {
+export default Ember.Controller.extend({
+  coverImageStyle: function() {
+    return (`background-image: url("${this.get('model.user.coverImageUrl')}")`).htmlSafe();
+  }.property('model.user.coverUrl'),
```

Comment:
It was wrong and wasn't even used :/ I've pushed another change removing a few of these, as well as cleanup.